pages/your_tasks/identifiers.md

```diff
@@ -65,11 +65,11 @@ If the research institute or group has a centralized and structured system (such
+When all records and measurements have been collected and you are ready to share your entire dataset with others, it is good practise to assign **globally unique persistent identifiers** in order to make your dataset more FAIR. "A Globally Unique Identifier (GUID) is a unique number that can be used as an identifier for anything in the universe and the uniqueness of a GUID relies on the algorithm that was used to generate it" ([What is a GUID?](http://guid.one/guid)). “A persistent identifier (PID) is a long-lasting reference to a resource. That resource might be a publication, dataset or person. Equally it could be a scientific sample, funding body, set of geographical coordinates, unpublished report or piece of software. Whatever it is, the primary purpose of the PID is to provide the information required to reliably identify, verify and locate it. A PID may be connected to a set of metadata describing an item rather than to the item itself" ([What is a persistent identifier, OpenAIRE](https://www.openaire.eu/what-is-a-persistent-identifier)).This means that any dataset with a PID will be findable even if the location of the dataset and its web address (URL) changes. The central registry that manage PID will ensure that the given PID will point you to the digital resource's current location. There are [different types of PID](https://en.wikipedia.org/wiki/Persistent_identifier), such as DOI, PURL, Handle and URN. [GO FAIR organisation](https://www.go-fair.org/fair-principles/f1-meta-data-assigned-globally-unique-persistent-identifiers/) provides examples of GUID, PID and services that supply identifiers.
 
 ### Considerations
 
-PIDs are essential to make your digitale object (datasets or resources) citable, enabling you to claim and receive credit for your research output. In turn, when you reuse someone else research output, you have to cite it.
+PIDs are essential to make your digital object (datasets or resources) citable, enabling you to claim and receive credit for your research output. In turn, when you reuse someone else research output, you have to cite it.
 
 There are a couple of different ways by which you can obtain a globally unique persistent identifier, and you need to decide which one is the best solution for your dataset or resource.
```

Comment:
Suggested revision: _"There are different ways to obtain a globally unique persistent identifier, and you..."_